### DISCUSSION TITLE
Enforce MU mod and allow downloading for maps that song get DMCA takedown.

### DISCUSSION THREAD

**Discussion by astro-angelfish** [ppy/osu#30558] [Ideas]
Like the title. I think we could allow downloading the beatmap without audio file that got DMCA takedown. And when playing the beatmap marked DMCA takedown, we could enforce the MU mod enabled, playing with metronome and enjoy the beatmap without the song.

I don't know if there is any legal problems if we only play with metronome, we already took down the song, and beatmap is credited to their mappers and should be shared to everyone as mappers want.

**Comment by ziv-vy** [comment]
1) DMCAs are rare.
 2) As soon as the DMCA is sent, the beatmap is hidden from search. That's a good thing. Nobody wants a map that is a literal metronome.
 3) Muted does **not** exist in Stable.
 4) If people want to play a beatmap, they will just pirate it.  As soon as a beatmap is taken down, the comments section gets flooded with download links.
 5) Muted still plays some of the music.
